BountyBoard.Core/Data/Achievement.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BountyBoard.Core.Data
+{
+    public class Achievement : DatabaseObject
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public byte[] Image { get; set; }
```

Comment:
in the database, i've found that it's better leaving it as a byte[]. i'm open to ideas